<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Incarnation editor v0.1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="centralWidgetLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QGridLayout" name="mainGridLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QTabWidget" name="menuTabWidget">
        <property name="tabPosition">
         <enum>QTabWidget::West</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <widget class="QWidget" name="homeTab">
         <attribute name="icon">
          <iconset resource="../images.qrc">
           <normaloff>:/icons/icons/Home.png</normaloff>:/icons/icons/Home.png</iconset>
         </attribute>
         <attribute name="title">
          <string/>
         </attribute>
         <attribute name="toolTip">
          <string>Home</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="2">
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="1" rowspan="3">
             <widget class="Line" name="line">
              <property name="frameShadow">
               <enum>QFrame::Plain</enum>
              </property>
              <property name="lineWidth">
               <number>2</number>
              </property>
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="newProjectButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string>New : Create an empty project</string>
              </property>
              <property name="text">
               <string>New project</string>
              </property>
              <property name="icon">
               <iconset resource="../images.qrc">
                <normaloff>:/icons/icons/NewFile.png</normaloff>:/icons/icons/NewFile.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>64</width>
                <height>64</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QPushButton" name="copyProjectButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string>Copy : Create a new project (from an existing project)</string>
              </property>
              <property name="text">
               <string>Copy project</string>
              </property>
              <property name="icon">
               <iconset resource="../images.qrc">
                <normaloff>:/icons/icons/CopyFile.png</normaloff>:/icons/icons/CopyFile.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>64</width>
                <height>64</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="2" column="2" colspan="2">
             <widget class="QListWidget" name="previousProjectsListWidget">
              <property name="enabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="openProjectButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string>Open : Open an existing project</string>
              </property>
              <property name="text">
               <string>Open project</string>
              </property>
              <property name="icon">
               <iconset resource="../images.qrc">
                <normaloff>:/icons/icons/OpenFile.png</normaloff>:/icons/icons/OpenFile.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>64</width>
                <height>64</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="1" column="2" colspan="2">
             <widget class="QLabel" name="recentProjectsLabel">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Recent projects&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="textFormat">
               <enum>Qt::AutoText</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QListWidget" name="projectsTemplatesListWidget">
              <item>
               <property name="text">
                <string>Model 1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Model 2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Model 3</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Templates&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="timelineTab">
         <attribute name="icon">
          <iconset resource="../images.qrc">
           <normaloff>:/icons/icons/Time.png</normaloff>:/icons/icons/Time.png</iconset>
         </attribute>
         <attribute name="title">
          <string/>
         </attribute>
         <attribute name="toolTip">
          <string>Timeline editor</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <layout class="QGridLayout" name="timelineTabLayout"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="structureTab">
         <attribute name="icon">
          <iconset resource="../images.qrc">
           <normaloff>:/icons/icons/Plan.png</normaloff>:/icons/icons/Plan.png</iconset>
         </attribute>
         <attribute name="title">
          <string/>
         </attribute>
         <attribute name="toolTip">
          <string>Structure editor</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="0">
           <layout class="QGridLayout" name="structureTabLayout"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="roomTab">
         <attribute name="icon">
          <iconset resource="../images.qrc">
           <normaloff>:/icons/icons/Room.png</normaloff>:/icons/icons/Room.png</iconset>
         </attribute>
         <attribute name="title">
          <string/>
         </attribute>
         <attribute name="toolTip">
          <string>Room editor</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QGridLayout" name="roomTabLayout"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="settingsTab">
         <attribute name="icon">
          <iconset resource="../images.qrc">
           <normaloff>:/icons/icons/Settings.png</normaloff>:/icons/icons/Settings.png</iconset>
         </attribute>
         <attribute name="title">
          <string notr="true"/>
         </attribute>
         <attribute name="toolTip">
          <string>Settings</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="1">
           <layout class="QGridLayout" name="settingsTabLayout">
            <item row="1" column="0">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="4">
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="settingsSaveButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Save</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="3">
             <widget class="QPushButton" name="settingsCancelButton">
              <property name="text">
               <string>Cancel</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="5">
             <widget class="QTabWidget" name="settingsTabsWidget">
              <property name="currentIndex">
               <number>2</number>
              </property>
              <widget class="QWidget" name="generalSettingsTab">
               <attribute name="title">
                <string>General</string>
               </attribute>
               <layout class="QGridLayout" name="gridLayout_5">
                <item row="0" column="0">
                 <layout class="QGridLayout" name="generalSettingsTabLayout">
                  <item row="1" column="3">
                   <widget class="QToolButton" name="workspaceToolButton">
                    <property name="text">
                     <string notr="true">...</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QLabel" name="workspaceLabel">
                    <property name="text">
                     <string>Workspace : </string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLabel" name="userLanguageLabel">
                    <property name="text">
                     <string>Language :</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                    <property name="margin">
                     <number>2</number>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="0">
                   <spacer name="horizontalSpacer_4">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item row="1" column="2">
                   <widget class="QLineEdit" name="workspaceLineEdit"/>
                  </item>
                  <item row="0" column="4">
                   <spacer name="horizontalSpacer_5">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item row="0" column="2" colspan="2">
                   <widget class="QComboBox" name="userLanguageComboBox">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="currentText">
                     <string notr="true">English</string>
                    </property>
                    <item>
                     <property name="text">
                      <string notr="true">English</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string notr="true">Français</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="timelineEditorSettingsTab">
               <attribute name="title">
                <string>Timeline editor</string>
               </attribute>
              </widget>
              <widget class="QWidget" name="structureEditorSettingsTab">
               <attribute name="title">
                <string>Structure editor</string>
               </attribute>
               <layout class="QGridLayout" name="gridLayout_9">
                <item row="0" column="0">
                 <layout class="QGridLayout" name="structureEditorSettingsTabLayout">
                  <item row="0" column="1">
                   <widget class="QLabel" name="structureEditorZoomLabel">
                    <property name="text">
                     <string notr="true">Zoom : </string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="4">
                   <spacer name="horizontalSpacer_7">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item row="0" column="0">
                   <spacer name="horizontalSpacer_6">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item row="0" column="2">
                   <widget class="QSlider" name="structureEditorZoomSlider">
                    <property name="minimum">
                     <number>1</number>
                    </property>
                    <property name="maximum">
                     <number>200</number>
                    </property>
                    <property name="value">
                     <number>100</number>
                    </property>
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="invertedAppearance">
                     <bool>false</bool>
                    </property>
                    <property name="invertedControls">
                     <bool>false</bool>
                    </property>
                    <property name="tickPosition">
                     <enum>QSlider::NoTicks</enum>
                    </property>
                    <property name="tickInterval">
                     <number>25</number>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="3">
                   <widget class="QLabel" name="structureEditorZoomValueLabel">
                    <property name="text">
                     <string>100%</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="roomEditorSettingsTab">
               <attribute name="title">
                <string>Room editor</string>
               </attribute>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
